examples/serverless-api/cerebrium.toml

```diff
@@ -0,0 +1,19 @@
+[cerebrium.deployment]
+name = "serverless-api"
+python_version = "3.11"
+docker_base_image_url = "debian:bookworm-slim"
+include = "[./*, main.py, cerebrium.toml]"
+exclude = "[.*]"
+
+[cerebrium.hardware]
+cpu = 2
+memory = 12.0
```

Comment:
Isn't 12GB RAM a bit excessive for this? I've never heard of Cerebrium but I'll try to check it out to test the PR.

Reply:
Sorry you are right - thats the default when I created the project